project/AppDependencies.scala

```diff
@@ -1,46 +1,31 @@
 import sbt._
+import play.core.PlayVersion
 
 object AppDependencies {
-  import play.core.PlayVersion
 
-  val compile = Seq(
+  private lazy val compile = Seq(
     play.sbt.PlayImport.ws,
-    "org.reactivemongo" %% "play2-reactivemongo"            % "0.20.13-play28",
-    "uk.gov.hmrc"       %% "play-frontend-hmrc"             % "1.1.0-play-28",
-    "uk.gov.hmrc"       %% "domain"                         % "6.2.0-play-28",
-    "com.typesafe.play" %% "play-json-joda"                 % "2.9.2",
-    "uk.gov.hmrc"       %% "play-conditional-form-mapping"  % "1.9.0-play-28",
-    "uk.gov.hmrc"       %% "bootstrap-frontend-play-28"     % "5.24.0",
-    "uk.gov.hmrc"       %% "tax-year"                       % "1.1.0"
+    "uk.gov.hmrc.mongo"       %% "hmrc-mongo-play-28"             % "0.73.0",
+    "uk.gov.hmrc"             %% "play-frontend-hmrc"             % "3.32.0-play-28",
```

Comment:
we are reverting back "play-frontend-hmrc" to its current version as we discovered some bugs and testing effort is too big to focus on it now, please revert this back to "1.1.0-play-28"